chrome/locale/en-US/zotero/preferences.ftl

```diff
@@ -99,3 +99,12 @@ preferences-advanced-custom-data-dir =
 preferences-advanced-default-data-dir =
     .value = (Default: { $directory })
     .aria-label = Default location
+
+preferences-sync-reset-restore-to-server-body = { -app-name } will replace data in “{ $libraryName }” on { $domain } with data from this computer. { $remoteItemsDeletedCount ->
+    [0] { "" }
+    *[other] { $remoteItemsDeletedCount } { $remoteItemsDeletedCount -> 
+        [1] item
+        *[other] items
+    } in the online library will be deleted!
+}
```

Comment:
Probably better to do this as a separate string that gets added after two newlines if >0.

And I think we can omit the exclamation mark. Arguably appropriate for thousands of items, but it might say "1 item", and in general it's probably better to be matter-of-fact about it.